wizard/src/utils/tmcf_generation.ts

```diff
@@ -0,0 +1,145 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  CsvData,
+  MappedThing,
+  Mapping,
+  MappingType,
+  MappingVal,
+} from "../types";
+
+const FIXED_CSV_TABLE = "CSVTable";
+const SVOBS_TYPE = "StatVarObservation";
+const PLACE_TYPE = "Place";
+const DCID_PROP = "dcid";
+const VALUE_PROP = "value";
+const MAPPED_THING_TO_SVOBS_PROP = new Map<MappedThing, string>([
+  [MappedThing.PLACE, "observationAbout"],
+  [MappedThing.STAT_VAR, "variableMeasured"],
+  [MappedThing.DATE, "observationDate"],
+  [MappedThing.UNIT, "unit"],
+  [MappedThing.VALUE, VALUE_PROP],
+]);
+
+//
+// A set of node/pv generation helpers
+//
+
+function initNode(idx: number, type: string): Array<string> {
+  const pvs = Array<string>();
+  pvs.push("Node: E:" + FIXED_CSV_TABLE + "->E" + idx.toString());
```

Comment:
maybe we can use template strings here and elsewhere where we are concatenating strings: 
```
'Node: E:${FIXED_CSV_TABLE}->E${idx.toString()}`
```

Reply:
Very nice!